components/IscnRegisterForm.vue

```diff
@@ -850,6 +866,10 @@ export default class IscnRegisterForm extends Vue {
     return 'CreativeWork'
   }
 
+  get defaultLanguage() {
+    return this.epubMetadata?.language || 'zh'
+  }
```

Comment:
Should set `this.language` as `defaultLanguage` on `mounted()`

Reply:
Also would be nice if we can set default as `en` or `zh` according to if we have non-ascii charater in `title` or `description`

Reply:
Use @watch to detect name & description language for book.

[🚸 Auto detect language for book](https://github.com/likecoin/app-like-co/pull/471/commits/6af5f50ddf8443855607eca376d6ba4e8c009f99)